source/_integrations/zha.markdown

```diff
@@ -423,7 +423,25 @@ In this theoretical example, a CC2652-based Zigbee coordinator has three CC2530
 
 In practice, you will likely need to add a lot more Zigbee router devices than in this example in order to extend the coverage of the network to reach that many devices.
 
-## Binding and unbinding
+## Zigbee Groups, Bindings and Unbindings
+
+ZHA integration support for native Zigbee Groups, as well as Zigbee Bindings and Unbindings. These features can be used separately or combined.
+
+### Zigbee Group
+
+A "Zigbee Group" enables native on-device grouping of multiple Zigbee lights, switches, and fans that enable controlling all compatible entities for those devices in those groups with only one command/entity.
+
+<div class='note'>
+Note that using a native "Zigbee Group" inside the ZHA integration instead of Home Assistant's [Group](/integrations/group/) integration should make for much faster response times and better synchronization, especially when grouping lights.
```

Comment:
This is actually something that isn't intuitive and may actually be harmful: using Zigbee groups improves the visual responsiveness (as in, devices will toggle at the same time) but it is only recommended if you control your devices about once or twice a minute, as Zigbee group commands are broadcasts and can only be sent every few seconds. Any more often will cause the group command to be rejected by the firmware.

Reply:
Addedinformation about that.